pkg/config/config_darwin.go

```diff
@@ -15,3 +15,11 @@ func customConfigFile() (string, error) {
 func ifRootlessConfigPath() (string, error) {
 	return rootlessConfigPath()
 }
+
+var defaultHelperBinariesDir = []string{
+	// FIXME: What paths can we use on mac?
```

Comment:
`/usr/libexec` seems to exist on Mac OS, so we could make that the first entry.

Reply:
Yes I have no idea about what paths can be used for mac or where homebrew install this ATM. That's why I marked this as draft.

Reply:
At the moment, Homebrew installs in `/usr/local/bin` or `/opt/homebrew/bin`, but the general consensus seems that we no longer want `gvproxy` to be in the executable path.  The next-best option seems to be to have Homebrew leave it in-key at `${HOMEBREW_PREFIX}/Cellar/podman/${version}/libexec`, but you can see that that version in the middle of the keg means we'll just need to put out a `containers.conf` file with the configured directory as part of the Homebrew install anyway.  So, for the moment, I'd not worry about the Mac for the default list -- what's here LGTM for defaults.

Which leads me to the search path for `containers.conf` ... reviewing that now.

Reply:
I reread (again) the Homebrew Formula Cookbook and discovered "the opt prefix".  If we add `/usr/local/opt/pod man/libexec`, `/opt/homebrew/bin`, and `/opt/homebrew/opt/podman/lib exec` to the defaults list, we _should_ be okay.

Reply:
@Luap99 Are you good to add the 3 directories above or shall I file a PR against your repo?

Reply:
I will add them, thanks for the list.